lib/open3.rb

```diff
@@ -953,43 +959,62 @@ def pipeline_rw(*cmds, &block)
   end
   module_function :pipeline_rw
 
-  # Open3.pipeline_r starts a list of commands as a pipeline with a pipe
-  # which connects to stdout of the last command.
-  #
-  #   Open3.pipeline_r(cmd1, cmd2, ... [, opts]) {|last_stdout, wait_threads|
-  #     ...
-  #   }
+  # :call-seq:
+  #   Open3.pipeline_r([env, ] *cmds, options = {}) ->[last_stdout, wait_threads]
```

Comment:
Nit: there's a space missing.

```suggestion
  #   Open3.pipeline_r([env, ] *cmds, options = {}) -> [last_stdout, wait_threads]
```